content/blog/2020/08/camel-quarkus-release-1.0.0/index.md

```diff
@@ -0,0 +1,131 @@
+
+## What is Camel Quarkus?
+
+Camel Quarkus brings the outstanding integration capabilities of Apache Camel to [Quarkus](https://quarkus.io/) - the
+toolkit for writing subatomically small and supersonically fast Java, Kotlin and Scala applications. In addition to
+memory consumption and start time improvements on stock JVMs, Quarkus also allows for compiling applications to
+[native executables](https://camel.apache.org/camel-quarkus/latest/user-guide/first-steps.html#_package_and_run_the_application)
+thus improving the performance characteristics even more.
+
+Camel Quarkus is suitable not only for creating long living services and potentially short living serverless
+applications but also for
+[command-style applications](https://camel.apache.org/blog/2020/07/command-line-utility-with-camel-quarkus/)
+that execute some specific task (or wait for some specific condition) and exit.
+
+## Developer joy
+
+Writing integrations has never been more joyful for developers than with Camel Quarkus:
+
+* Easy to start with: either via [code.quarkus.io](https://code.quarkus.io/) or using `mvn quarkus:create`
+* The "live coding" feature, a.k.a. `mvn compile quarkus:dev` shortens your development iterations - check
+  [this 2 mins video](https://www.youtube.com/watch?v=4lXSf8DBQkQ)
+* Easy to test using JUnit 5 extensions
+* Hides the complexity of the native compilation using [GraalVM](https://www.graalvm.org/) or
+  [Mandrell](https://github.com/graalvm/mandrel)
+
+Check our https://camel.apache.org/camel-quarkus/latest/user-guide/first-steps.html[User guide]!
+
+## Camel components coverage
+
+Since the [early beginnings](https://github.com/quarkusio/quarkus/tree/0.11.0/extensions/camel) the number of supported
+Camel bits went up substantially. As of Quarkus 1.0.0 and
+Camel 3.4, Camel Quarkus supports:
+
+* 155 out of 350 Camel components (44%)
+* 27 out of 43 Camel data formats (63%)
+* 13 out of 17 Camel languages (76%)
+* 17 out of 42 miscellaneous Camel components (40%)
+
+Some of these are supported in JVM mode only - please refer to the
+[complete list](https://camel.apache.org/camel-quarkus/latest/list-of-camel-quarkus-extensions.html) for details.
+
+## Bootstrapping methods
+
+There are two ways to bootstrap and configure Camel Quarkus:
+
+1. Bare
+2. With Camel Main
+
+Both methods configure and start a Camel Context for you. The main difference is in how much the resulting Camel Context
+obeys the conventions known from Camel on other platforms. If you like to configure your routes using
+`camel.component.*` properties, then you need to add the `camel-quarkus-main` dependency and the properties
+will work as usual on Camel standalone or Camel SpringBoot. Of course, this brings some overhead. You may prefer this
+option if you used Camel before or if you are migrating from other platform.
+
+If you rather come from the Quarkus side and you never wrote Camel integrations before, you may choose to use the bare
```

Comment:
I would say from CDI side - its not really Quarkus side. CDI is a bit "coding heavy" in terms of configuration as this requires Java coding and adding annotations and whatnot.

It reads as if it is two "big" alternatives, that you cannot use CDI with Camel Main, and vice versa.

Reply:
> I would say from CDI side

Good idea.

> It reads as if it is two "big" alternatives, that you cannot use CDI with Camel Main, and vice versa.

Good point, the text should not awake that impression.

Reply:
> > It reads as if it is two "big" alternatives, that you cannot use CDI with Camel Main, and vice versa.
> 
> Good point, the text should not awake that impression.

Adding `You can configure Camel using CDI even if you have Camel Main in your application.`